### PR TITLE
⚒️ exclude 0 value from floorprice

### PR DESCRIPTION
**Pull request by roiLeo**
Will it blend too?

### CODE REVIEW

File: src/resolvers/series.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class SeriesResolver {
         COUNT(distinct ne.current_owner) as sold, 
         COUNT(ne.*) as total, 
         AVG(ne.price) as average_price, 
-        MIN(ne.price) as floor_price, 
+        MIN(NULLIF (ne.price, 0)) as floor_price,
```
Comment (vikiival) [inline src/resolvers/series.ts:26]:
```suggestion
        MIN(NULLIF(ne.price, 0)) as floor_price, 
```

**Review by vikiival — APPROVED** [review]
LGTM

**Comment by vikiival** [comment]
The question is: Do we need this? Does it break UI somehow?

**Comment by roiLeo** [comment]
> The question is: Do we need this? Does it break UI somehow?

Well, floor_price has a great chance to be at 0 (new listed nft see series stats), it would help to query (sort) HIGHEST/LOWEST floor_price

**Comment by vikiival** [comment]
release 06.01.2022 inb4 9pm

**Pull request merged**